sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateDao.java

```diff
@@ -114,23 +116,23 @@ public int countUsers(DbSession session, PermissionQuery query, Long templateId)
     }
   }
 
-  public int countGroups(DbSession session, PermissionQuery query, long templateId) {
+  public int countGroups(DbSession session, OldPermissionQuery query, long templateId) {
     return countGroups(session, query, templateId, null);
   }
 
-  private static int countGroups(DbSession session, PermissionQuery query, long templateId, @Nullable String groupName) {
+  private static int countGroups(DbSession session, OldPermissionQuery query, long templateId, @Nullable String groupName) {
     Map<String, Object> parameters = groupsParameters(query, templateId);
     if (groupName != null) {
       parameters.put("groupName", groupName.toUpperCase(Locale.ENGLISH));
     }
     return mapper(session).countGroups(parameters);
   }
 
-  public boolean hasGroup(DbSession session, PermissionQuery query, long templateId, String groupName) {
+  public boolean hasGroup(DbSession session, OldPermissionQuery query, long templateId, String groupName) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)